test/tasks/solvers.test.ts

```diff
@@ -88,10 +89,31 @@ describe("Task: solvers", () => {
       expect(await authenticator.isSolver(solver.address)).to.be.true;
       await run("solvers", {
         subtask: "remove",
-        solver: solver.address,
+        address: solver.address,
         printTransaction: true,
       });
       expect(await authenticator.isSolver(solver.address)).to.be.true;
     });
   });
+
+  describe("setManager", () => {
+    it("sets the manager", async () => {
+      expect(await authenticator.manager()).not.to.equal(notManager.address);
+      await run("solvers", {
+        subtask: "setManager",
+        address: notManager.address,
+      });
+      expect(await authenticator.manager()).to.equal(notManager.address);
+    });
+
+    it("does not set the manager when --print-transaction is specified", async () => {
+      expect(await authenticator.isSolver(notSolver.address)).to.be.false;
+      await run("solvers", {
+        subtask: "add",
+        address: notSolver.address,
+        printTransaction: true,
+      });
+      expect(await authenticator.isSolver(notSolver.address)).to.be.false;
+    });
```

Comment:
Is this intentionally a test again to add solvers?

Reply:
Indeed a mistake, fixed!